resource/gomega.go

```diff
@@ -10,8 +10,8 @@ import (
 
 func matcherToGomegaMatcher(matcher interface{}) (types.GomegaMatcher, error) {
 	switch x := matcher.(type) {
-	case string, int, bool, float64:
-		return gomega.Equal(x), nil
+	case string, int, uint, int64, uint64, bool, float64:
+		return gomega.BeEquivalentTo(x), nil
```

Comment:
JSON deserialization engine usually parses value to `int`, which can't be compared with `uint64` via `gomega.Equal` matcher. `gomega.BeEquivalentTo` is less strict version that wors in this case.